### PR TITLE
LINK-1622  | Don't send sensitive data to Sentry

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
- Always use `useHandleError` hook when sending error reports to Sentry to keep error handling consistent.
- Send only username instead of full user object to sentry 
- Omit `admin_users`, `registration_admin_users` and `regular_users` from `organization` payload when sending error report to Sentry
- Omit `first_name`, `last_name`, `email`, `date_of_birth`, `phone_number` and `extra_info` from `signup` or `signup group` payload when sending error report to Sentry
- Omit `registration_user_accesses` from `registration` payload when sending error report to Sentry
- Omit `name` and `email` from `feedback` payload when sending error report to Sentry
- Omit `user_email`, `user_name` and `user_phone_number` from `event` payload when sending error report to Sentry

## Closes
[LINK-1622](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1622)

[LINK-1622]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1622?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/domain/organization/organizationForm/OrganizationForm.tsx [hunk 1]
```diff
@@ -85,65 +68,28 @@ const OrganizationForm: React.FC<OrganizationFormProps> = ({
...
     useOrganizationServerErrors();
 
-  const [createOrganizationMutation] = useCreateOrganizationMutation();
-
+  const onCreate = async (values: OrganizationFormFields) => {
+    await createOrganization(values, {
+      onError: (error: ServerError) => showServerErrors({ error }),
+      onSuccess: goToOrganizationsPage,
+    });
+    // await createOrganization(values, {
```
Comment (mikkojamG) [inline src/domain/organization/organizationForm/OrganizationForm.tsx:84]: This comment can probably be removed?

**Review by mikkojamG — CHANGES_REQUESTED** [review]

File: src/domain/organization/organizationForm/OrganizationForm.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/organization/organizationForm/OrganizationForm.tsx]: True that. I removed the unnecessary comment

File: src/domain/signup/utils.ts [hunk 2]
```diff
@@ -88,3 +91,15 @@ export const getSignupGroupInitialValuesFromSignup = (
     signups: [getSignupInitialValues(signup)],
   };
 };
+
+export const omitSensitiveDataFromSignupPayload = (
+  payload: SignupInput | UpdateSignupMutationInput
+): Partial<SignupInput | UpdateSignupMutationInput> =>
+  omit(payload, [
```
Comment (harriris-vincit) [inline src/domain/signup/utils.ts:98]: For the back-end exceptions, also these fields were included in the data scrubbing rules:
```
city
street_address
zipcode
membership_number
native_language
service_language
```
These should probably unified.

Comment (jorilindell) [reply src/domain/signup/utils.ts:98]: Good point. I unified the data. Also created a PR to Linked Registration UI to make it happen there as well. https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/92

**Review by harriris-vincit — APPROVED** [review]
Looks good

**Pull request merged**